### PR TITLE
Design a New York Times Clone

### PR DESCRIPTION
**Pull request by kagaramag**
## Title
This is PR serves at designing a clone of NewYork Times article page.

## Description 
The project is for practising positioning and Ffoating in HTML and CSS.

## Screenshot
image

### CODE REVIEW

**Review by fatymahmed — CHANGES_REQUESTED** [review]
## Hi @djallas @Angelesh1995 
Your project looks good 👏👏, however, there are a few things that you need to fix.

**Navigation bar**
- [ ]  The NY Times logo should be a link.

**Bottom part**
- [ ] The _More in space and astronomy_ section should contain **links** to other articles with image previews on the leftmost side, kindly change these to links.

**Footer**
- [ ] It should have a fixed position(it sticks to the bottom part of the screen).

**Implementation**
- [ ] You should use `<aside>` tag for Editor's picks section on the rightmost side,  see [screenshot from Pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/aside_requirement.png)

- [ ] Kindly add code linter stickler to your repo, check [this](https://github.com/microverseinc/linters-config/tree/master/css) on how to do it.
- [ ] Your Pull Request should have a short description, see [screenshot from Pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png).

[OPTIONAL]
- To easily achieve absolute positioning for the editor's picks and subscribe sections, you can use the `<section>` and `<aside>` tags which help keep the document flow.

**Review by fatymahmed — CHANGES_REQUESTED** [review]
### Kindly make the suggested changes and request for another code review.
### Status: Changes Required
_Feel free to reach out to me on slack (Fatima) if you have any questions, I'd be happy to help._

**Review by fatymahmed — COMMENTED** [review]
## Hi @djallas @Angelesh1995,
Great job on making some of the changes requested, please make all changes for this project to be approved.

**Bottom part**

- The _More in space and astronomy_ section should contain links to other articles with image previews on the leftmost side, kindly change these to links. You've made the title to a link, but each section under this should be a link (the image and the title), I've attached an image that contains marked areas of what I'm specifying in case it's not clear.
screenshot


-  Your Pull Request should have a short description, see [screenshot from Pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png), remove the image and include a short description of the content you have so far.

**Review by fatymahmed — CHANGES_REQUESTED** [review]
### Kindly make the suggested changes and request for another code review.
### Status: Changes Required
_Feel free to reach out to me on slack (Fatima) if you have any questions, I'd be happy to help._

**Review by nikasulo — CHANGES_REQUESTED** [review]
# Really great work @djallas and @Angelesh1995 :rocket: 
Your project looks very neat, your markup is semantic and you have put in a lot of visible work and paid attention to detail :handshake: congrats :medal_military: . 

Did you by any chance get round to seeing the comments @fatymahmed rightfully made about the links in the section for which she attached the screenshot? The small change she suggested hasn't been implemented. Please consider looking into her comments! :100: 

##### Think I missed something? Need some help with this or any of your projects? Please feel free to reach out to me on [Twitter](https://twitter.com/oluwadamilareo_) or [Instagram](https://instagram.com/oluwadamilare_olusakin), even if you just want to chat and I will respond to you as soon as I can.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,845 @@
...
+          </h4>
+          <div class="more-article-box">
+            <a href="https://www.nytimes.com/2020/01/30/science/nasa-spitzer-space-telescope.html?action=click&module=MoreIn&pgtype=Article&region=Footer&action=click&module=MoreInSection&pgtype=Article&region=Footer&contentCollection=Space%20and%20Astronomy">
+            <div class="image">
+              <img src="https://static01.nyt.com/images/2020/02/04/science/30SCI-SPITZER1c/30SCI-SPITZER1c-threeByTwoSmallAt2X-v2.jpg" alt="image">
+              </div>
+              <div class="text-right platform">NASA/JPL</div>
+              <h5 class="no-padding no-margin">NASA’s Spitzer Space Telescope Ends 16-Year Mission of Discovery</h5>
+              <div class="text-gray timestamp">Jan. 30</div>
+            </a>
```
Comment (kagaramag) [inline index.html]: Hello @Oluwadamilareolusakin, I have gotten your feedback mentioning that we should add links in the section. I implemented the feedback @fatymahmed suggested us. Can you kindly take a look on codes I highlighted here? Let us know if we are doing it right or if we are missing something. Thank you

**Comment by Anna-Myzukina** [comment]
## Hi team!
### Nice job! Project looks! :+1: 

But you still need to fix next:

#### Bottom part

*  It should contain links to other articles with image previews on the leftmost side 



* Check HTML markup with this validator https://validator.w3.org/#validate_by_upload. (Students were asked to use this one, so in most cases their code should be already validated) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/validation_requirement.png) from Pathwright)
screen

errors


### Please kindly fix the above issues and send another code review request.
* If you have any questions or need help, you can message me on slack (@ Anna Muzykina). Happy coding

File: index.html [hunk 2]
```diff
@@ -0,0 +1,845 @@
...
+          </h4>
+          <div class="more-article-box">
+            <a href="https://www.nytimes.com/2020/01/30/science/nasa-spitzer-space-telescope.html?action=click&module=MoreIn&pgtype=Article&region=Footer&action=click&module=MoreInSection&pgtype=Article&region=Footer&contentCollection=Space%20and%20Astronomy">
+            <div class="image">
+              <img src="https://static01.nyt.com/images/2020/02/04/science/30SCI-SPITZER1c/30SCI-SPITZER1c-threeByTwoSmallAt2X-v2.jpg" alt="image">
+              </div>
+              <div class="text-right platform">NASA/JPL</div>
+              <h5 class="no-padding no-margin">NASA’s Spitzer Space Telescope Ends 16-Year Mission of Discovery</h5>
+              <div class="text-gray timestamp">Jan. 30</div>
```
Comment (Anna-Myzukina) [inline index.html]: Only content inside tag h5 should be link
Example: 

Screenshot from 2020-02-03 21-12-03

File: index.html [hunk 3]
```diff
@@ -0,0 +1,845 @@
...
+            </a>
+          </div>
+          <div class="more-article-box">
+            <a href="https://www.nytimes.com/2020/01/29/science/daniel-inouye-solar-telescope-pictures.html?action=click&module=MoreIn&pgtype=Article&region=Footer&action=click&module=MoreInSection&pgtype=Article&region=Footer&contentCollection=Space%20and%20Astronomy">
+            <div class="image">
+              <img src="https://static01.nyt.com/images/2020/01/31/science/31conversation/full_image_high-threeByTwoSmallAt2X.jpg" alt="image">
+            </div>
+            <div class="text-right platform">Image by NSO/NSF/AURA</div>
+            <h5 class="no-padding no-margin">NASA’s Spitzer Space Telescope Ends 16-Year Mission of Discovery</h5>
+            <div class="text-gray timestamp">Jan. 29</div>
```
Comment (Anna-Myzukina) [inline index.html]: Only content inside tag h5 should be link

**Review by Anna-Myzukina — CHANGES_REQUESTED** [review]
The project still needs changes

**Comment by kagaramag** [comment]
_**To whom it may concern**_
@Angelesh1995 and I implemented the feedback provided by @fatymahmed and @Anna-Myzukina.
Regarding the feedback on **_`More in space and astronomy`_** section
Links are moved to wrap images(so that they can be clickable) and also inside h5 tags

**Review by nikasulo — APPROVED** [review]
# Congrats! :rocket:
Please merge and keep moving :medal_military: See you at the top!

##### Think I missed something? Need some help with this or any of your projects? Please feel free to reach out to me on [Twitter](https://twitter.com/oluwadamilareo_) or [Instagram](https://instagram.com/oluwadamilare_olusakin), even if you just want to chat and I will respond to you as soon as I can.

**Pull request merged**